src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.OSX/CertificateData.cs

```diff
@@ -471,6 +471,9 @@ private static string FindAltNameMatch(byte[] extensionBytes, GeneralNameType ma
                         case DerSequenceReader.DerTag.UTF8String:
                             value = tavReader.ReadUtf8String();
                             break;
+                        case DerSequenceReader.DerTag.T61String:
+                            value = tavReader.ReadT61String();
```

Comment:
A test should be added for this new code.  Preferably one which doesn't require dual licensing.  If nothing else, it should be relatively easy to craft a certificate using the CertificateRequest API.